### ISSUE TITLE
[Feature Request] Flix View for Landscape Content and Square Content and Episode List View

### ISSUE THREAD

**Issue by Goldenfreddy0703** [nfm886/skin.cosmic#59]
Hey so its been a few weeks sense i reviewed your skin but i just tested it out today and everything looks amazing and i like how you did the spotlights for stuff cause it looks amazing. Some problems was that the scrolling text is not scrolling for some views and that scrolling text is not working for main menu. 2 Suggestions i have is adding a Flix View for Landscape Content and Square Content and having an Episode List View like Auramod. Again love your skin and really loved the improvements you have added. Just needs a little bit of more views but again, everything looks great. 

Vertical Widgets View - Looks correct with posters
![image](https://user-images.githubusercontent.com/62456796/132927632-7f2335fd-c727-4575-8701-f2fbbc99c5f6.png)

Horizontal Widgets View - Does not look correct with landscape content
![image](https://user-images.githubusercontent.com/62456796/132927590-ddc52db9-7bda-4499-9f75-4b8e08c3e8c7.png)

Episode List View from Auramod
![image](https://user-images.githubusercontent.com/62456796/132927707-f60080d0-7c0c-46d3-9238-e5df3a401fe8.png)

**Comment by rijsab** [comment]
I like the look of Auramod as well.. :D

**Comment by icpryde** [comment]
That auramod episode list view is sexy!

**Comment by Goldenfreddy0703** [comment]
> That auramod episode list view is sexy! 

I can definitely agree.

**Comment by nfm886** [comment]
> Just needs a little bit of more views but again, everything looks great.

Yeah, I know. I'm planning to add more views and remove legacy ones but I don't know when it happens.

**Comment by Goldenfreddy0703** [comment]
Nice thank you very much and your the best.

**Comment by nfm886** [comment]
My plans for near future are to change PVR osd (I found a look I really like :D) and improve Flix Spotlight a little bit.
Then I will add new view types and remove legacy ones.

**Comment by Goldenfreddy0703** [comment]
Ooo nice plan, i can't wait in the future.

**Comment by heppen7** [comment]
Overdone. Room for new issues after new update.
